static/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -446,10 +446,6 @@ const FeatureListItem = styled('span')`
 `;
 
 const Description = styled('div')`
-  font-size: 1.5rem;
-  line-height: 2.1rem;
-  margin-bottom: ${space(2)};
```

Comment:
Current font-size is 15px. This is not standard, we should use 16px (1rem in the new scale) instead. Also, the line-height and margin-bottom are redundant - they don't affect the UI by much.

With new size:
<img width="396" alt="Screen Shot 2021-12-16 at 11 39 04 AM" src="https://user-images.githubusercontent.com/44172267/146445521-da773688-eefc-4b49-8bff-ca554dc473e4.png">